src/App.js

```diff
@@ -1,5 +1,9 @@
 import Logo from './logo.png';
 import FacebookLogo from './fb_icon.svg';
+import { DateTime, Settings } from 'luxon'
+
+Settings.defaultZoneName = 'America/Denver'
```

Comment:
This is to make sure that not matter where in the world you're viewing the site from, it'll always show the pickups by Denver time. This is because "happening today" would not be accurate if you're viewing from NY at 1am NY time.